services/motion/builtin/builtin_test.go

```diff
@@ -940,66 +1019,6 @@ func TestCheckPlan(t *testing.T) {
 	})
 }
 
-func TestArmGantryPlanCheck(t *testing.T) {
```

Comment:
Why are we removing this test?

Reply:
I chose to remove since we do not support checking a plan for an arm at the moment. I can the test add back in but it seems a little strange to be testing for functionality we do not have.

Reply:
Looking at the test, it appears we do support it, no? In the sense that given an arm+gantry plan, some obstacles, and the current inputs, it can indeed correctly determine whether or not that plan now intersects the obstacles. That seems like a useful thing to check.

That said this test is in no way related to the built-in motion service, and the built-in motion service does not, as you said, support checking arm plans.

Rather than moving entirely this test should be able to be moved essentially-unchanged to the `motionplan` package.